apps/settings/js/icc.js

```diff
@@ -61,6 +59,18 @@ require([
     var _backHandler = null;
     init();
 
+
+    function localizeElement(element, l10n) {
+      if (typeof(l10n) === 'string') {
+        element.setAttribute('data-l10n-id', l10n);
+      } else if (l10n.raw) {
+        element.removeAttribute('data-l10n-id');
+        element.textContent = l10n.raw;
+      } else {
+        navigator.mozL10n.setAttributes(element, l10n.id, l10n.attrs);
+      }
+    }
+
```

Comment:
Sorry I didn't follow the evolution of l10n framework too much. Could you explain a bit the logic here? I don't understand why we need this function and what `l10n.raw` is. Thanks.